.github/workflows/review.yml

```diff
@@ -26,23 +26,23 @@ jobs:
           python-version: ${{ matrix.python }}
 
       - name: Run tests
-        
+
         # Python 3.6 tests have been removed since swagger-spec-validator no longer supports it.
         # A successful workflow run for Python 3.6 is required by the GitHub branch protection rules.
         if: ${{ matrix.python != 3.6 }}
-        
+
         env:
           PYTHON_VERSION: ${{ matrix.python }}
         run: tox -e $(tox -l | grep py${PYTHON_VERSION//.} | paste -sd "," -)
 
       - name: Report coverage
-        if: ${{ matrix.python == 3.9 }}
+        if: ${{ matrix.python != 3.6 }}
         run: |
           pip install coverage
           coverage report
 
       - name: Check for incompatibilities with publishing to PyPi
-        if: ${{ matrix.python == 3.8 }}
+        if: ${{ matrix.python == 3.11 }}
```

Comment:
In the `publish.yml` workflow we use python3.8 to run twine so we should match these versions. Can you edit your PR to make the publish workflow use python3.11 too please?